tests/bluetooth/test_handle_prompt.py

```diff
@@ -14,29 +14,86 @@
 #  limitations under the License.
 
 import pytest
-from test_helpers import execute_command, goto_url, subscribe, wait_for_event
+from test_helpers import (AnyExtending, execute_command, goto_url,
+                          send_JSON_command, subscribe, wait_for_event)
+
+HTML_SINGLE_PERIPHERAL = """
+<div>
+    <a href="#" id="bluetooth" target="_blank">bluetooth</a>
+    <script>
+        var options = {filters: [{name:"SomeDevice"}]};
+        document.getElementById('bluetooth').addEventListener('click', () => {
+        navigator.bluetooth.requestDevice(options);
+    });
+    </script>
+</div>
+"""
 
 
 @pytest.mark.asyncio
-async def test_bluetooth_handle_prompt(websocket, context_id):
-    await subscribe(websocket, ["bluetooth"])
-
-    await goto_url(websocket, context_id,
-                   "http://127.0.0.1:8080/deviceRequest.html")
-
-    # await execute_command(
-    #   websocket, {
-    #       "method": "script.callFunction",
-    #       "params": {
-    #           "functionDeclaration": """() => {
-    #               document.querySelector("#bluetooth").click();
-    #             }""",
-    #           "target": {
-    #               "context": context_id
-    #           },
-    #           "awaitPromise": True,
-    #           "userActivation": True
-    #       }
-    #   })
-
-    await wait_for_event(websocket, "bluetooth.requestDevicePromptOpened")
+@pytest.mark.parametrize('capabilities', [{
+    'goog:chromeOptions': {
+        'args': ['--enable-features=WebBluetooth']
```

Comment:
I think it would still override the default args.

Reply:
Right: https://github.com/GoogleChromeLabs/chromium-bidi/pull/2582

Reply:
@OrKoN and I might need to rebase our branches to pick up that fix?

Reply:
> @OrKoN and I might need to rebase our branches to pick up that fix?

yes, you have to rebase